charts/osm-arc/Chart.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
+name: osm-arc
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.5.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+appVersion: v0.5.0
+
+dependencies: 
+- name: osm
+  version: "0.5.0-rc.1"
+  repository: "file://../../charts/osm"
```

Comment:
Should this be https://github.com/openservicemesh/osm/tree/main/charts/osm?

Reply:
@michelleN I noticed we dont have index.yaml in https://github.com/openservicemesh/osm/tree/main/charts/osm is that going to be an issue finding older versions of the osm chart?